x11/auto-multiple-choice/Portfile

```diff
@@ -87,7 +89,8 @@ depends_run             \
                         port:texlive \
                         port:texlive-fonts-extra \
                         port:texlive-lang-japanese \
-                        port:texlive-latex-extra
+                        port:texlive-latex-extra \
+                        port:xorg-server
```

Comment:
Most ports do not add such a dependency even when they need it. However, a few do for some reason. You must allow for XQuartz or the -devel version to be installed in any case. 

`path:bin/Xquartz:xorg-server
`

Reply:
We can't even do that; someone could, for example, just be remote displaying to another box and have no X server at all and the port could still be of use. So, I'd remove the X server dependency entirely.

Reply:
This port need a X11 server and doesn't work without one. There was a discussion recently on  the [macports-users list](https://lists.macports.org/pipermail/macports-users/2018-November/046147.html) where someone said that only the macports version will be maintained. So I think it's time to add X11 server as a dependance.

Adding the X11 server dependancy also simplify the installation process. As this port is mainly used by teachers who often aren't at ease with command line and tool dependancies this will help installation instructions.

I've tested the build with "path:bin/Xquartz:xorg-server" instead of "port:xorg-server" and it worked fine in macOS 10.13.6 and 10.14.2.

Now I've to find how to push the change…

Reply:
"This port need a X11 server and doesn't work without one" — but the X server doesn't need to be on the same machine.

Reply:
It only adds one more step to tell them to install the xorg-server port. It can even be included in the install command:

`sudo port install xorg-server auto-multiple-choice
`

Admittedly, that may not work if the user already has XQuartz installed. But requiring xorg-server(-devel) does not help that issue and may even hinder it.